<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcDialog</class>
 <widget class="QDialog" name="ProcDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Processing...</string>
  </property>
  <widget class="QPlainTextEdit" name="pteCalibProcWindow">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>371</width>
     <height>131</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PshBtnCalibFinish">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="LblCalibYML">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>167</red>
         <green>167</green>
         <blue>167</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>167</red>
         <green>167</green>
         <blue>167</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>167</red>
         <green>167</green>
         <blue>167</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>167</red>
         <green>167</green>
         <blue>167</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>  Status</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="PrgBarCalibStatus">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
